packages/server/utils/AzureDevOpsServerManager.ts

```diff
@@ -452,6 +510,52 @@ class AzureDevOpsServerManager {
     return oAuthRes
   }
 
+  async addScoreComment(
+    instanceId: string,
+    dimensionName: string,
+    finalScore: string,
+    meetingName: string,
+    discussionURL: string,
+    remoteIssueId: string,
+    projectKey: string
+  ) {
+    const comment = `*${dimensionName}: ${finalScore}*
+    \n[See the discussion | ${discussionURL}] in ${meetingName}
+
+    \n_Powered by [Parabol | ${ExternalLinks.GETTING_STARTED_SPRINT_POKER}]_`
```

Comment:
-1 we're using markdown here but it looks like the comment is just a string in Azure:

![comment-style](https://user-images.githubusercontent.com/39854876/166952291-1277dcad-92f0-4341-915d-7453f37f22f4.png)

We try to keep PRs as small as possible so would you mind creating a separate issue to make these comments look more similar to our comments in the other integrations? For example, here's GitHub's:

![Screenshot 2022-05-05 at 16 01 59](https://user-images.githubusercontent.com/39854876/166952995-f96ca922-9697-455a-861b-876696c2beaf.png)

Reply:
We created issue https://github.com/ParabolInc/parabol/issues/6509.

We will resolve this in another PR as requested.  SHouldn't take more than an hour or so.